docs/index.md

```diff
@@ -50,17 +51,83 @@ steampipe plugin install zoom
 
 ### Credentials
 
-| Item | Description |
-| - | - |
-| Credentials | [Create an SDK App](https://marketplace.zoom.us/docs/guides/build/sdk-app) to get the API key and secret. |
-| Permissions | SDK / JWT apps can access all Zoom APIs. |
-| Radius | Each connection represents a single Zoom account. |
-| Resolution |  1. `api_key` and `api_secret` in Steampipe config.<br />2. `ZOOM_API_KEY` and `ZOOM_API_SECRET` environment variables. |
+| Item        | Description                                                                                                                                                                                                           |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Credentials | [Create a Server-to-Server OAuth app](https://developers.zoom.us/docs/internal-apps/create/) to get the Account ID, Client ID and Client Secret.                                                                      |
+| Permissions | Server-to-Server OAuth apps or SDK / JWT apps can access all Zoom APIs.                                                                                                                                               |
```

Comment:
Can scopes be set on OAuth or JWT apps now?

Reply:
Yes, we can set the scope on OAuth or JWT apps.